app/src/main/java/com/example/moviedb/data/model/Movie.kt

```diff
@@ -0,0 +1,116 @@
+package com.example.moviedb.data.model
+
+import android.os.Parcel
+import android.os.Parcelable
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import com.google.gson.annotations.Expose
+import com.google.gson.annotations.SerializedName
+
+@Entity(tableName = "movie")
+data class Movie(
+    @SerializedName("id")
+    @Expose
+    @PrimaryKey
+    val id: String = "",
+
+    @SerializedName("overview")
+    @Expose
+    val overview: String = "",
+
+    @SerializedName("original_language")
+    @Expose
+    val originalLanguage: String = "",
+
+    @SerializedName("original_title")
+    @Expose
+    val originalTitle: String = "",
```

Comment:
Trừ id ra những cái khác để nullable vì server có thể trả về giá trị null (id để not null vì nó là primary key trong room database)